lib/contextual_logger.rb

```diff
@@ -5,94 +5,98 @@
 require_relative './contextual_logger/context/handler'
 
 module ContextualLogger
-  def self.new(logger)
-    logger.extend(self)
+  class << self
+    def new(logger)
+      logger.extend(LoggerMixin)
+    end
   end
 
+  module LoggerMixin
   def global_context=(context)
```

Comment:
looks like all this needs to be indented now that's it's part of a module

Reply:
I intentionally left it unindented so the PR diff would show the net changes only. If it's served that purpose, I can indent it now.

Reply:
Changed and pushed.